tests/OmniSharp.Roslyn.CSharp.Tests/CompletionFacts.cs

```diff
@@ -2220,6 +2220,39 @@ public static void Main()
             }
         }
 
+        [Theory]
+        [InlineData("dummy.cs")]
+        [InlineData("dummy.csx")]
+        public async Task TestOverrideWithTrailingWhitespacePrior(string filename)
+        {
+            const string input = @"
+namespace OmniSharp.Cake
```

Comment:
would you mind changing this to a generic Foo or something of that sort like the rest of the tests? someone in the future will surely spend time wondering what is Cake specific about this (in fact, that was my first automatic reaction too)